test/configCases/output-module/issue-16040/index.js

```diff
@@ -1,8 +1,6 @@
 import foo from "./foo.js";
 import bar from "./bar.js";
 
-console.log(foo + bar);
```

Comment:
Not a good due to only `console.log`, I think in future we need refactor tests (step by step) to avoid using `console.log` to check that code is works without any problems